app/controllers/ops_controller/settings/zones.rb

```diff
@@ -112,7 +113,7 @@ def zone_set_record_vars(zone, mode = nil)
     zone.settings[:proxy_server_ip] = @edit[:new][:proxy_server_ip]
     zone.settings[:concurrent_vm_scans] = @edit[:new][:concurrent_vm_scans]
 
-    zone.update_authentication({:windows_domain => {:userid => @edit[:new][:userid], :password => @edit[:new][:password]}}, :save => (mode != :validate))
+    zone.update_authentication({:windows_domain => {:userid => @edit[:new][:userid], :password => @edit[:new][:password]}}, {:save => (mode != :validate)})
```

Comment:
This one looks like a code change - not a Rubocop thing.  Can you clarify why this change is needed.

Reply:
I don't think it's a code change. I just added `{}` around a hash (second argument) to silence the following rubocop warning:
```
app/controllers/ops_controller/settings/zones.rb:115:125: C: Style/BracesAroundHashParameters: Missing curly braces around a hash parameter.
    zone.update_authentication({:windows_domain => {:userid => @edit[:new][:userid], :password => @edit[:new][:password]}}, :save => (mode != :validate))
                                                                                                                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```